### PR TITLE
Remove unpackable decorator, use asdict()

### PR DESCRIPTION
**Pull request by mdbenito**
Inspired by #232, this PR removes `@unpackable` and uses `dataclasses.asdict()` instead

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -15,4 +15,4 @@ ray[default] >= 0.8
 tox<4.0.0
 tox-wheel
 types-tqdm
-twine
+twine==4.0.2
```
Comment (AnesBenmerzoug) [inline requirements-dev.txt:18]: Is this necessary?

**Review by AnesBenmerzoug — CHANGES_REQUESTED** [review]
Looks good to me! it's just missing a changelog entry and merging the develop branch into it.

File: requirements-dev.txt (hunk 1 above)
Comment (mdbenito) [reply requirements-dev.txt:18]: It was in my branch for some reason. I don't remember, but I guess I had some problem?

**Comment by mdbenito** [comment]
> Looks good to me! it's just missing a changelog entry and merging the develop branch into it.

How do you "request changes"?? and how do I mark said changes as done? I have tried resolving the conversation about twine, but it does not affect the "Changes requested" 🤔🤔

**Review by AnesBenmerzoug — APPROVED** [review]
I think I have to approve it in order for the review status to change.

**Pull request merged**